recipes/cli-duckdb/meta.yaml

```diff
@@ -0,0 +1,79 @@
+{% set name = "cli-duckdb" %}
```

Comment:
duckdb-cli sounds like a more common name

Reply:
Agreed, but all the other modules are named `{package}-duckdb`: https://anaconda.org/search?q=duckdb

Reply:
I think the other packages are integrations of duckdb with other tools, e.g. intake-duckdb it to allow intake to use ducked, not the other way around.

I think calling it duckdb-cli makes more sense, and the extensions would also be switched, e.g. duckdb-httpfs